dataplug-facebook/app/com/hubofallthings/dataplugFacebook/apiInterfaces/FacebookFeedUploadInterface.scala

```diff
@@ -94,7 +94,7 @@ class FacebookFeedUploadInterface @Inject() (
 
 object FacebookFeedUploadInterface {
   val defaultApiEndpoint = ApiEndpointCall(
-    "https://graph.facebook.com/v5.0",
+    "https://graph.facebook.com/v9.0",
```

Comment:
"https://graph.facebook.com/v9.0" - seems like these strings should be in a config file